packages/alignments/src/SNPCoverageTrack/configSchema.js

```diff
@@ -58,7 +49,7 @@ export default pluginManager => {
 
       defaultRendering: {
         type: 'stringEnum',
-        model: types.enumeration('Rendering', ['snpcoverage']),
+        model: types.enumeration('Rendering', ['snpcoverage', 'svg']),
```

Comment:
what is svg here

Reply:
Svg was included because PileupTrack could be chosen between pileup and svg defaultRendering and since they share the same parent config editor svg would be applied to SNPCoverageTrack which would error. 

Better solution that will be implemented is to remove defaultRendering field from SNPCoverageTrack since it only has one renderer anyways.

This may need addressing in future if subtrack concept is expanded upon and the subtracks each have multiple renderers to select from and do not overlap. Will probably have to change the defaultRendering name to be subtrack specific aka defaultPileupRendering or similar idea